src/views/UserSettings.vue

```diff
@@ -113,12 +113,12 @@ export default {
   methods: {
     initUserSettingsForm() {
       this.userSettingsForm.currency = this.appStore.user.last_currency_used
-      this.userSettingsForm.selectedLanguage = this.languageList.find(lang => lang.code === localeManager.guessDefaultLocale()) || this.languageList.find(lang => lang.code === 'en')
+      this.userSettingsForm.selectedLanguage = this.languageList.find(lang => lang.code === localeManager.guessDefaultLocale()).code
```

Comment:
it feels strange that the `guessDefaultLocale` happens in the settings form (vs on app startup).

here it means the new user arrived on the app, the app will show up by default as 'en', right ? even if the user has a 'French' browser ? And after logging in, and going to its settings page, then the form field language will show 'French' :thinking: 

Or I'm missing something ? :sweat_smile: